integration_tests/integration_tests.py

```diff
@@ -45,20 +45,21 @@
             'nochip': True
         }
     },
-    'semantic_segmentation.basic': {
-        'task': 'semantic_segmentation',
-        'module': 'integration_tests.semantic_segmentation',
-        'kwargs': {
-            'nochip': False
-        }
-    },
-    'semantic_segmentation.nochip': {
-        'task': 'semantic_segmentation',
-        'module': 'integration_tests.semantic_segmentation',
-        'kwargs': {
-            'nochip': True
-        }
-    }
+    # Semantic segmentation tests currently failing, disabled until fixed
+    # 'semantic_segmentation.basic': {
+    #     'task': 'semantic_segmentation',
+    #     'module': 'integration_tests.semantic_segmentation',
+    #     'kwargs': {
+    #         'nochip': False
+    #     }
+    # },
+    # 'semantic_segmentation.nochip': {
+    #     'task': 'semantic_segmentation',
+    #     'module': 'integration_tests.semantic_segmentation',
+    #     'kwargs': {
+    #         'nochip': True
+    #     }
+    # }
```

Comment:
These tests are failing on `master` as well. I've disabled them for now and created an issue to track fixing them later.

Reply:
I saw these got fixed in #1236 so I dropped the commit and rebased onto `master`.

Reply:
@lewfish Heads up, I did run into a test failure here for `semantic_segmentation.basic`. Log lines in question are [here](https://github.com/azavea/raster-vision/runs/3284883787#step:3:5042).

Weirdly, it passed just fine on a subsequent run. Just wanted to raise that to your attention in case you run into flakey results later on.